### PR TITLE
Fix VA Deletion Check in E2E Testing

### PR DESCRIPTION
**Pull request by razo7**
Check with Consistently instead of Eventually and use offset

### CODE REVIEW

File: test/e2e/far_e2e_test.go [hunk 1]
```diff
@@ -405,7 +405,7 @@ func wasNodeRebooted(nodeName string, nodeBootTimeBefore time.Time) {
 
 // checkVaDeleted verifies if the va has already been deleted due to resource deletion
 func checkVaDeleted(va *storagev1.VolumeAttachment) {
-	EventuallyWithOffset(1, func() bool {
+	ConsistentlyWithOffset(1, func() bool {
```
Comment (slintes) [inline test/e2e/far_e2e_test.go:408]: Why this change? For waiting until something happens, Eventually is the right tool 🤔

File: test/e2e/far_e2e_test.go [hunk 2]
```diff
@@ -455,12 +455,12 @@ func checkRemediation(nodeName string, nodeBootTimeBefore time.Time, oldPodCreat
 	By("Getting new node's boot time")
 	wasNodeRebooted(nodeName, nodeBootTimeBefore)
 
-	By("checking if old VA has been deleted")
-	checkVaDeleted(va)
-
 	By("checking if old pod has been deleted")
 	checkPodDeleted(pod)
 
+	By("checking if old volume attachment has been deleted")
```
Comment (slintes) [inline test/e2e/far_e2e_test.go:461]: Why the change in order? AFAIK in a real world the pod won't be deleted before its related VA is deleted (that's why we delete the VA actually). So first checking the VA makes more sense to me.

**Comment by razo7** [comment]
Closing the PR as it is not needed anymore:
1. Marc's remarks are correct
2. VA deletion by FAR could interfere with the process of detaching the VAs by kube-control-manager.
